clang/test/3C/tree.c

```diff
@@ -1,6 +1,6 @@
-// RUN: 3c -base-dir=%S -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
-// RUN: 3c -base-dir=%S %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
-// RUN: 3c -base-dir=%S %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: 3c --extra-arg="-D_FORTIFY_SOURCE=0" -base-dir=%S -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
```

Comment:
Maybe a comment in the file about why we need `-D_FORTIFY_SOURCE=0` (linking to this PR or to #630 for the full explanation) would be in order too.